### PR TITLE
Added view according to feedback, refactored code and fixed bugs

### PR DESCRIPTION
**Pull request by senseihimanshu**

### CODE REVIEW

File: api/storage/csv/employee.csv [hunk 1]
```diff
@@ -2,4 +2,8 @@ employeeId,email,name,designation,role
 100,senseihimanshu@gmail.com,Himanshu Sharma,intern,interviewer
 101,nangchetnamaw@gmail.com,Chetna Mongmaw,intern,interviewer
 105,deepak.yadav@gmail.com,Deepak Yadav,intern,interviewer
-106,faizan.akhtar@gmail.com,Faizan Akhtar,intern,interviewer
+106,sankalp.shastri@gmail.com,Sankalp Shastri,intern,interviewer
+107,sonali.chawla@gmail.com,Sonali Chawla,intern,interviewer
+108,lord.rama@gmail.com,Lord Rama,manager,superuser
```
Comment (sonalichawla16) [inline api/storage/csv/employee.csv:7]: no need to commit to this file

File: web/src/app/app-routing.module.ts [hunk 2]
```diff
@@ -1,58 +1,56 @@
...
-    {
-      path: "", redirectTo: "home", pathMatch: "full"
-    },
-    {
-      path: "home", component: HrComponent
-    }
-  ]},
+  {
+    path: "su", component: AppNavBarComponent, canActivate: [RoleGuardService], data: { role: "admin" }, children: [
+      {
```
Comment (sonalichawla16) [inline web/src/app/app-routing.module.ts:17]: su ???

File: api/storage/csv/employee.csv (hunk 1 above)
Comment (senseihimanshu) [reply api/storage/csv/employee.csv]: Done!

File: web/src/app/app-routing.module.ts (hunk 2 above)
Comment (senseihimanshu) [reply web/src/app/app-routing.module.ts:17]: Done!

**Pull request merged**